debian/package.json

```diff
@@ -10,18 +10,20 @@
     "player": "./bin/player-window",
     "production": "NODE_ENV=production electron .",
     "build": "NODE_ENV=production NEW_VERSION=$npm_package_version ./bin/build",
-    "rebuild": "electron-rebuild"
+    "install": "electron-rebuild"
```

Comment:
hmm overriding `install` is not recommended. better use `postinstall` https://docs.npmjs.com/misc/scripts#examples